src/features/command-palette/command-palette.ts

```diff
@@ -82,16 +84,15 @@ export class CommandPalette extends AbstractUIExtension {
         const selectedElements = toArray(root.index.all().filter(e => isSelectable(e) && e.selected));
         if (selectedElements.length === 1) {
             const firstElement = selectedElements[0];
-            if (isBoundsAware(firstElement)) {
-                const viewport = findParentByFeature(firstElement, isViewport);
-                if (viewport) {
-                    x += (firstElement.bounds.x + firstElement.bounds.width - viewport.scroll.x) * viewport.zoom;
-                    y += (firstElement.bounds.y - viewport.scroll.y) * viewport.zoom;
-                } else {
-                    x += firstElement.bounds.x + firstElement.bounds.width;
-                    y += firstElement.bounds.y;
-                }
+            if (firstElement) {
```

Comment:
Why should `selectedElements[0]` ever be undefined?

Reply:
Oh, right, good catch. Thanks!